src/bfti.c

```diff
@@ -130,7 +142,16 @@ static void processdir(void * passv)
     // push subdirectories into the queue
     //descend(passmywork, dir, in.max_level);
 
-    if ((db=opendb(passmywork->name,3,0))) {
+    SNPRINTF(dbname, MAXPATH, "%s/%s", passmywork->name, DBNAME);
+    if ((db=opendb(dbname, RDONLY, 1, 1,
```

Comment:
Need to insert create_tables function?

Reply:
It turns out that that is correct, since the `createtables` argument is 0, so only opening the database has to happen.